src/endpoints/process_models/process_model_controller.ts

```diff
@@ -58,7 +58,7 @@ export class ProcessModelController {
 
   public async startProcessInstance(request: HttpRequestWithIdentity, response: Response): Promise<void> {
     const processModelId: string = request.params.process_model_id;
-    const startEventId: string = request.params.start_event_id;
+    const startEventId: string = request.query[restSettings.queryParams.startEventId];
```

Comment:
This won't work, because the value for that reference is `:start_event_id`.
See: 
https://github.com/process-engine/management_api_contracts/blob/develop/src/rest_settings.ts#L9

However, `request.query` stores this parameter as `start_event_id`.

So you'll have to do this: 

```suggestion
    const startEventId: string = request.query.startEventId;
```

Reply:
Hmm, but we basically do the same thing with the `end_event_id` parameter here: https://github.com/process-engine/management_api_http/blob/develop/src/endpoints/process_models/process_model_controller.ts#L62

Reply:
But if you look closely, you'll find that this is pretty much the only occurrence in all the APIs.

Frankly, I'm surprised that line you posted there hasn't caused trouble yet. 😳 

In fact, that line should be changed asap as well.